Tests/SwiftGenKitTests/AssetCatalogTests.swift

```diff
@@ -16,11 +16,30 @@ class AssetCatalogTests: XCTestCase {
     XCTDiffContexts(result, expected: "empty.plist", sub: .xcassets)
   }
 
-  func testDefaults() throws {
+  func testImages() throws {
     let parser = AssetsCatalogParser()
     try parser.parse(path: Fixtures.path(for: "Images.xcassets", sub: .xcassets))
 
     let result = parser.stencilContext()
-    XCTDiffContexts(result, expected: "defaults.plist", sub: .xcassets)
+    XCTDiffContexts(result, expected: "images.plist", sub: .xcassets)
+  }
+
+  func testColors() throws {
```

Comment:
Add a third test for an asset catalog mixing colors + images, generating an heterogeneous context, to ensure that's a working use case too.